views/timelisting.tmpl

```diff
@@ -17,27 +17,32 @@
                 {{$pending := (subtract $lastGrouping .Offset)}}
                 {{$dropVal := $lastGrouping}}
                 {{if gt $lastGrouping 200}}{{$dropVal = 200}}{{end}}
-                <span class="h2 d-flex pt-4 pb-1 pr-2">{{.TimeGrouping}}</span>
+                <span class="h2 d-flex pt-4 pb-1 pr-2">
+                    {{.TimeGrouping}}
+                    <span class="dcricon-info fs24 ml-3 mt-2"></span>
+                </span>
 
                 <div class="pb-1 d-flex justify-content-end align-items-end flex-wrap">
                   <span class="fs12 nowrap text-secondary px-2 my-2">
                       {{intComma (add .Offset 1)}} &ndash; {{intComma $oldest}} of {{ intComma $lastGrouping }} rows
                   </span>
-                  <span class="fs12 nowrap text-right">
+                  <span class="fs12 nowrap text-right mr-2">
                       <select
                           data-target="pagenavigation.pagesize"
                           data-action="change->pagenavigation#setPageSize"
                           data-offset="{{$.Offset}}"
                           data-offsetkey="offset"
                           class="dropdown text-secondary my-2 {{if lt $pending 10}}disabled{{end}}"
-                          {{if lt $pending 10}}disabled="disabled"{{end}}
+                          {{if lt $pending 10}}disabled='disabled'{{end}}
                       >
-                      {{if ge $pending 20}}<option {{if eq $count 20}}selected{{end}} value="20">20 per page</option>{{end}}
-                      {{if ge $pending 30}}<option {{if eq $count 30}}selected{{end}} value="30">30 per page</option>{{end}}
-                      {{if ge $pending 50}}<option {{if eq $count 50}}selected{{end}} value="50">50 per page</option>{{end}}
-                      {{if ge $pending 100}}<option {{if eq $count 100}}selected{{end}} value="100">100 per page</option>{{end}}
-                      {{if eq $dropVal $count 20 30 50 100}}{{else}}<option value="{{$dropVal}}">{{$dropVal}} per page</option>{{end}}
-                      {{if eq $count 20 30 50 100 200}}{{else}}<option selected value="{{$count}}">{{$count}} per page</option>{{end}}
+                        <option {{if eq $count 10}}selected{{end}} value="10">10 per page</option>
+                        <option {{if eq $count 20}}selected{{end}} value="20">20 per page</option>
+                        <option {{if eq $count 30}}selected{{end}} value="30">30 per page</option>
+                        <option {{if eq $count 40}}selected{{end}} value="40">40 per page</option>
+                        <option {{if eq $count 50}}selected{{end}} value="50">50 per page</option>
+                        {{if eq $count 10 20 30 40 50}}{{else}}
+                            <option selected value="{{$count}}">{{$count}} per page</option>
+                        {{end}}
```

Comment:
You've changed the behavior here. For instance, right now, on the months view, there is an option to show 50 per page, but Decred only has 45 months of history. What was the reasoning for removing the filters here?

Reply:
@buck54321 I literally copy pasted the designs

Reply:
The dropdown has only these options

Reply:
reverting back original behavior